pkg/gcc/static_threshold.go

```diff
@@ -0,0 +1,17 @@
+package gcc
```

Comment:
Should this be moved into `overuse_detector_test.go` I don't see it used anywhere else? Do we plan on using it more/is it a common idiom?